x/exchange/client/cli/query_setup_test.go

```diff
@@ -793,6 +793,7 @@ func TestSetupCmdQueryGetAccountCommitments(t *testing.T) {
 		expExamples: []string{
 			exampleStart + " " + cli.ExampleAddr,
 			exampleStart + " --account " + cli.ExampleAddr,
+			exampleStart + " --asset nash",
```

Comment:
After fixing the `AddQueryExample`, you'll need to fix it here too:
```suggestion
			exampleStart + " " + cli.ExampleAddr + " --asset nash",
```